Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,293 @@
+    final m_flow_nominal={mDis_flow_nominal,-mBui_flow_nominal,mByp_flow_nominal},
+    final dp_nominal=dp_nominal)
+    "Bypass junction"
+    annotation (Placement(transformation(extent={{-60,-270},{-40,-290}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTDisRet(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=mDis_flow_nominal)
+    "District return temperature sensor"
+    annotation (Placement(transformation(extent={{30,210},{50,190}})));
+  Buildings.Fluid.FixedResistances.Junction spl(
+    redeclare final package Medium=MediumSer,
+    final energyDynamics=energyDynamics,
+    final m_flow_nominal={mBui_flow_nominal,-mDis_flow_nominal,-mByp_flow_nominal},
+    final dp_nominal=dp_nominal)
+    "Bypass junction, splitter"
+    annotation (Placement(transformation(extent={{-60,190},{-40,210}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage conVal(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=mDis_flow_nominal,
+    final dpValve_nominal=dpConVal_nominal,
+    use_inputFilter=true,
+    riseTime(displayUnit="s") = 60)
+    "Control valve"
+    annotation (Placement(transformation(extent={{-10,210},{10,190}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiRet(redeclare final package
+      Medium = MediumSer, final m_flow_nominal=mBui_flow_nominal)
+    "Building return temperature sensor"
+    annotation (Placement(transformation(extent={{-220,210},{-200,190}})));
+  Modelica.Fluid.Valves.ValveIncompressible cheVal(
+    redeclare final package Medium=MediumSer,
+    final allowFlowReversal=false,
+    final dp_nominal=dpCheVal_nominal,
+    final m_flow_nominal=mByp_flow_nominal,
+    final filteredOpening=true,
+    riseTime(
+      displayUnit="s")=60,
+    final checkValve=true)
+    "Check valve (backflow preventer)"
+    annotation (Placement(transformation(extent={{10,-10},{-10,10}},
+        rotation=90,
+        origin={-50,-10})));
+  Modelica.Blocks.Math.Add dTdis(
+    final k1=-1,
+    final k2=+1)
+    "Temperature difference on the district side"
+    annotation (Placement(transformation(extent={{60,-114},{80,-94}})));
+  Modelica.Blocks.Math.Product pro
+    "Product"
+    annotation (Placement(transformation(extent={{120,-120},{140,-100}})));
+  Modelica.Blocks.Math.Gain cp(
+    final k=cp_default)
+    "Specific heat multiplier to calculate heat flow rate"
+    annotation (Placement(transformation(extent={{182,-120},{202,-100}})));
+  Modelica.Blocks.Continuous.Integrator int(
+    final k=1)
+    "Integration"
+    annotation (Placement(transformation(extent={{260,-160},{280,-140}})));
+  Modelica.Blocks.Interfaces.RealOutput Q_flow(
+    final quantity="Power",
+    final unit="W",
+    displayUnit="kW")
+    "Measured power demand at the ETS"
+    annotation (Placement(
+        transformation(extent={{300,-130},{340,-90}}), iconTransformation(
+          extent={{300,-130},{340,-90}})));
+  Modelica.Blocks.Interfaces.RealOutput Q(
```

Comment:
when it is possible, please put all the interface instance, like RealInput/RealOutput, BooleanInput/BooleanOutput, Fulid port, etc. to be the section after parameters, before other types of blocks.

Reply:
Thank you for the suggestion! I have checked every model and make sure the right sequence.